src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,18 @@
 import React from 'react';
+import { MovieCard } from '../MovieCard';
 import './MoviesList.scss';
+import { Movie } from '../../types/Movie';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movieList: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movieList }) => (
+  <ul className="movies">
+    {movieList.map(item => (
+      <li key={item.imdbId}>
+        <MovieCard {...item} />
+      </li>
```

Comment:
```suggestion
      <MovieCard {...item} key={item.imdbId} />
```
move li to MovieCard component